comps/reranks/video-rag-qna/local_reranking.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import time
+
+from langsmith import traceable
```

Comment:
langsmith is removed from OPEA since it's not free for use.

Reply:
Thanks for your comment, removed.